images/UnicycleRide.svg

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Sounds good.  Would you like me to redo the icon? (I appreciate that this is a bit of a niche sport and you probably have more important things to work on :) ).

Thanks!

Reply:
Just in case it's helpful, I re-did the icons to more closely match the BikeRide icon proportions.  The commit is [31aba3a1f7b2c8304e5a6f87c40fa485de743565](https://github.com/mr-growl/ActivityTracker/commit/31aba3a1f7b2c8304e5a6f87c40fa485de743565)

Reply:
it's better now but the dimensions doesn't match other icons. sorry if I'm a bit paranoiac about this, but it will look odd if an icon is smaller then others. Also the blue background is already taken by the Run. I'd call just Unicycle, it's more clean (personally I'd call Bike instead of BikeRide but it will break things. Also, with a bike what can you do if not a ride?)
Since yesterday I started working on the icon I have now the finished work, please tell me if you like the icon and the color.

Reply:
That looks great!  Thanks for putting in the effort (again, this is a pretty niche sport so I appreciate the effort).

Reply:
You're welcome! well, I have a unicycle too (even if I don't make long rides around, just some easy trick :wink:) so I really appreciate your contribution!